docs/CHANGELOG.rst

```diff
@@ -12,6 +12,16 @@ Unreleased
 
 Added
 -----
+* Add `Non-canonical splice sites penalty`, `Disallow soft clipping`
+  and `Report alignments tailored specifically for Cufflinks`
+  parameters to hisat2 process
+
+================
```

Comment:
Should be under "Changed". Also, rebase will make an invalid merge of changelog. Keep an eye on that.